src/node.cc

```diff
@@ -515,6 +534,18 @@ inline void PlatformInit() {
   RegisterSignalHandler(SIGINT, SignalExit, true);
   RegisterSignalHandler(SIGTERM, SignalExit, true);
 
+  {
+    struct sigaction sa;
+    memset(&sa, 0, sizeof(sa));
+    sa.sa_sigaction = TrapWebAssemblyOrContinue;
+#ifndef __FreeBSD__
+    sa.sa_flags = SA_RESETHAND;
+#endif
+    sigfillset(&sa.sa_mask);
```

Comment:
I'm not saying this is wrong but are you sure you want to block all signals while the signal handler is running? A comment explaining the rationale for that would be good.